### PR TITLE
Update for v0.25

### PR DESCRIPTION
**Pull request by mdubus**

### CODE REVIEW

**Comment by curquiza** [comment]
When you are doing a WIP, open directly a draft PR, or convert your already existing PR as a draft :) 

Capture d’écran 2022-01-13 à 19 22 03

**Comment by mdubus** [comment]
> When you are doing a WIP, open directly a draft PR, or convert your already existing PR as a draft 🙂 No need to put WIP in the name of the PR See our contributing guidelines: https://github.com/meilisearch/gatsby-plugin-meilisearch/blob/main/CONTRIBUTING.md#github-pull-requests-
> 
> Capture d’écran 2022-01-13 à 19 22 03

Thanks a lot !
Since I was in a rush I totally forgot this option 🙈
Just did it now, thanks again ! 🙏

File: gatsby-node.js [hunk 1]
```diff
@@ -52,12 +52,14 @@ exports.onPostBuild = async function ({ graphql, reporter }, config) {
         })
 
         const index = client.index(currentIndex.indexUid)
-        await index.deleteIfExists()
+        if (index) await index.delete()
```
Comment (bidoubiwa) [inline gatsby-node.js:55]:
```suggestion
        await index.delete()
```

Index will always exists as it does not make any http call. If the index does not exists the `task` in MeiliSearch will fail but it does not impact the following tasks.

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
🔥

**Comment by bidoubiwa** [comment]
bors merge

**Review by bidoubiwa — APPROVED** [review]
LGTM

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**